src/SprykerEco/Zed/Heidelpay/Business/Mapper/QuoteToHeidelpayRequest.php

```diff
@@ -82,9 +84,43 @@ protected function mapQuoteInformation(QuoteTransfer $quoteTransfer, HeidelpayRe
                 ->setIdOrder($this->generateQuoteId($quoteTransfer))
         );
 
+        $customerRegistrationDate = $this->findCustomerRegistrationDate($quoteTransfer->getCustomer());
+
+        $heidelpayRequestTransfer->setRiskInformation(
+            (new HeidelpayRiskInformationTransfer())
+                ->setIsCustomerGuest((bool)$quoteTransfer->getCustomer()->getIsGuest())
+                ->setCustomerSince($customerRegistrationDate)
+                ->setCustomerId($quoteTransfer->getCustomer()->getIdCustomer())
+        );
+
         return $heidelpayRequestTransfer;
     }
 
+    /**
+     * @param \Generated\Shared\Transfer\CustomerTransfer $customerTransfer
+     *
+     * @return string|null
+     */
+    protected function findCustomerRegistrationDate(CustomerTransfer $customerTransfer): ?string
+    {
+        $createdAtDate = $customerTransfer->getCreatedAt();
+        $createdAtDateFormatted = $createdAtDate ? $this->formatDate($createdAtDate) : $createdAtDate;
+
+        return $createdAtDateFormatted;
+    }
+
+    /**
+     * @param string $date
+     *
+     * @return string
+     */
+    protected function formatDate(string $date): string
+    {
+        $dateArray = explode(' ', $date);
```

Comment:
I would prefer to have this in the proper way. E.g. smth like `(new \DateTime($date))->format('Y-m-d');`